src/containers/table/index.js

```diff
@@ -152,16 +152,21 @@ class TableComponent extends Component {
                                                   this.enableBulkSelect({ checked }, filterProps.data)
                                                 }
                                               />
-                                            ) : null}
+                                            </div>
+                                          </Table.HeaderCell>
+                                        ) : null}
+                                        {this.props.serialize && (
```

Comment:
this is a boolean field, rename accordingly. isShowSerialNumber

update readme regarding this prop